src/app/views/user/user-register/user-register.component.scss

```diff
@@ -1,72 +1,22 @@
 @import '../../../styles/colors';
 
-
 .register-container {
-
-  .register-form-outer-container {
-
-    .register-form-inner-container {
-      padding-top: 30px;
-      max-width: 500px;
-      margin: 0 auto;
-      padding-left: 20px;
-      padding-right: 20px;
-
-      .register-form-input-group-header {
-        margin-top: 30px;
-        h4 {
-          font-weight: 500;
-        }
-
-      }
-
-      .register-form-input-container {
-        margin-bottom: 0;
-        input {
-          border: 1px solid rgba(0, 0, 0, 0.15);
-        }
-
-        .register-form-input-description-text {
-          opacity: 0.5;
-          font-size: 12px;
-        }
-      }
-
-      .register-form-terms-container {
-        margin-top: 30px;
-      }
-
-      .register-form-button-container {
-        margin-top: 20px;
-        padding-bottom: 1.5em;
-        text-align: center;
-
-        button {
-          display: block;
-          margin: 15px auto 30px;
-          min-width: 250px;
-        }
-      }
-
-      .save-message {
-        color: green;
-      }
-
-      .error-message {
-        color: $color-cerise-500;
-      }
+  .register-form-button-container {
+    margin-top: 20px;
+    padding-bottom: 1.5em;
+    text-align: center;
+
+    .register-form-button {
+      margin: 15px auto 30px;
```

Comment:
Properties should be ordered min-width, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)